src/libs/Metrics/types.ts

```diff
@@ -0,0 +1,6 @@
+type Metrics = {
```

Comment:
What about changing `index.ts` and `index.native.ts` to follow this pattern we have been using?

```ts
const metrics: Metrics = {
    canCapturePerformanceMetrics: () => CONFIG.CAPTURE_METRICS,
    canCaptureOnyxMetrics: () => CONFIG.ONYX_METRICS,
};

export default metrics;
```

Reply:
This way `export` is changed to `export default`. I tried to avoid such changes, do you think it's safe to change?

Reply:
Sorry, it should be `export metrics;`, you are right 🙂

Reply:
oh ok 🤦 

What about this:

```ts
// types.ts
type CanCapturePerformanceMetrics = () => boolean;
type CanCaptureOnyxMetrics = () => boolean;

export type {CanCapturePerformanceMetrics, CanCaptureOnyxMetrics};

// index.ts
import CONFIG from '../../CONFIG';
import {CanCaptureOnyxMetrics, CanCapturePerformanceMetrics} from './types';

// We don't capture performance metrics on web as there are enough tools available
const canCapturePerformanceMetrics: CanCapturePerformanceMetrics = () => false;

const canCaptureOnyxMetrics: CanCaptureOnyxMetrics = () => Boolean(CONFIG.ONYX_METRICS);

export {canCaptureOnyxMetrics, canCapturePerformanceMetrics};
```

Since `canCapturePerformanceMetrics` and `canCaptureOnyxMetrics` has been treated as separate things we do the same for the types and avoid things like `Metrics['canCapturePerformanceMetrics']`.

Reply:
Looks good to me @fabioh8010, updated the PR.